data.tf

```diff
@@ -0,0 +1,9 @@
+locals {
+  monitoring_subnet_resource_id_slice = split("/", var.monitoring_subnet_id)
```

Comment:
nit: consider creating locals for the slice components we use instead of exposing the slice variable. Would be nice to resolve all of these in one place.